server/game/cards/01_SOR/YodaOldMaster.ts

```diff
@@ -0,0 +1,30 @@
+import AbilityHelper from '../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../core/card/NonLeaderUnitCard';
+import { TargetMode } from '../../core/Constants';
+
+export default class YodaOldMaster extends NonLeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: '4405415770',
+            internalName: 'yoda#old-master'
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addWhenDefeatedAbility({
+            title: 'Choose any number of players to draw 1 card',
+            targetResolver: {
+                mode: TargetMode.Select,
+                choices: (context) => ({
+                    //TODO: Make this selection prompt less awkward: create a way to select any number of options from an arbitrary list.
```

Comment:
Agreed. It looks like this is the only ability that allows you to select multiple players so there's not a strong need to create a special tool for this situation, but maybe more generally something that would do like checkboxes could be helpful in this and a few other situations. Good note to discuss with the FE folks at some point.